### DISCUSSION TITLE
Mysterious lines

### DISCUSSION THREAD

**Discussion by PolyLittle** [keiwando/toonsquid#33] [Q&A] [answered]
Hello everyone 🙂 

While working on my project I realized these strange Lines popping up for very short time. While working it wouldn't be a problem but they pop up as well after exporting. 
Does anyone have an idea where they come from and how to avoid them?

![image](https://user-images.githubusercontent.com/116555990/197733074-cc044389-6eab-40e5-b26b-2e2ac67c35e4.jpeg)

![image](https://user-images.githubusercontent.com/116555990/197733099-699d1448-1cb3-4ae1-b2c1-145f12556bb2.jpeg)

**Comment by keiwando** [comment, marked as answer]
Hi!
This definitely appears to be a rendering bug that I haven't seen before. I will try to fix this asap, especially since you say that it also appears in the export but I'm going to need some more information to hopefully help me narrow this down. Sorry for the inconvenience!

1. If at all possible, it would be very helpful if you could export the project and send it to me via contact@toonsquid.com so that I can take a look at what exactly might be causing this. (If it's too large to send directly via e-mail you can upload it to e.g. Google Drive and send me a link).
2. Which iPad model and operating system version are you using?
3. You say the lines only pop up very briefly and then disappear again, so the screenshots are probably from a screenrecording, right? Or do the lines stay on screen long enough for you to be able to take a regular screenshot of them?
4. Do they always appear in the same locations or on the same frames?

**Reply by PolyLittle** [reply]
Hey :)

1. I will send you the link to the youtubevideo via mail in a second
2. I'm using an iPad Pro 9,7 ". I think it's from 2016 ? Operating System is 15.7
3. I stoped the video but some lines stay even longer. It's different
4. Yes they do

**Reply by keiwando** [reply]
I have identified and fixed the issue and will submit an update (1.6.1) with the fix relatively soon.

**Reply by PolyLittle** [reply]
Awesome! and thank you very much for the help via email. Fastest support I'f seen for quite a while :)

**Reply by PolyLittle** [reply]
The the last patch solved the Problem. Thank you very much! :)
